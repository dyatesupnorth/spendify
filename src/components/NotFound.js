import React from "react";
import ReactDOM from "react-dom";
import { NavLink} from 'react-router-dom';
const NotFound = () => (
    <div>This is a 404! <NavLink to="/">Go home</NavLink></div>
)
export default NotFound;
