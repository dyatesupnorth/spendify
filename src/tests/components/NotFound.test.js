import React from 'react';
import {shallow} from 'enzyme';
import NotFound from '../../components/NotFound';

test('should render Not Found correctly', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
    //expect(wrapper.find('h1').length).toBe(1)
//   const renderer = new ReactShallowRenderer();
//   renderer.render(<Header />);
//   expect(renderer.getRenderOutput()).toMatchSnapshot();
});
