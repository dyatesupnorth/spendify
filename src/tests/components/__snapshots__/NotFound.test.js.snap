// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Not Found correctly 1`] = `
<div>
  This is a 404! 
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    to="/"
  >
    Go home
  </NavLink>
</div>
`;
